go.mod

```diff
@@ -89,7 +89,7 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/stoewer/go-strcase v1.1.0 // indirect
 	github.com/stretchr/testify v1.4.0
-	github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20200319153356-a183662640d9
+	github.com/terraform-providers/terraform-provider-aws v0.0.0
```

Comment:
What does switching the version like this do? just curious.

Reply:
go modules don't let you switch major versions. I'm unsure of how you would actually do this if you wanted to do it ;-) The way around this is to specify commit timestamp and hash. So that version string remains the same "v1.60.1-0.", 20200319153356 is the timestamp for commit hash a183662640d9". That locks in , in this case, v2.60.0.

Reply:
But to v0.0.0?

Reply:
I think you can put whatever you want there, since it just gets replaced down below in the replace block.